x-pack/plugins/uptime/server/lib/saved_objects.ts

```diff
@@ -32,6 +28,16 @@ export const umDynamicSettings: SavedObjectsType = {
       heartbeatIndices: {
         type: 'keyword',
       },
+      certificatesThresholds: {
+        properties: {
+          errorState: {
+            type: 'long',
```

Comment:
Kind of nit but `long` might not be necessary. Effectively an unimportant observation though as we're only going to have one of these objects per space.